lsp-dotty.el

```diff
@@ -34,15 +34,15 @@
        )))
 
 (defun lsp-dotty--get-root ()
-  (lsp-make-traverser
-   #'(lambda (dir)
-       (directory-files dir nil ".sbt"))))
+  (lsp-make-traverser "project"))
```

Comment:
BTW this changes how root folder is resolved. Previous version was not working for me for some reason